test_vm/tests/verified_claim_test.rs

```diff
@@ -59,14 +64,14 @@ fn verified_claim_scenario() {
         v.get_epoch() + max_prove_commit_duration(&Policy::default(), seal_proof).unwrap();
     let deal_term_min = 180 * EPOCHS_IN_DAY;
 
-    let deal_size = PaddedPieceSize(32u64 << 30);
+    let deal_size = 32u64 << 30;
     let deals = market_publish_deal(
         &v,
         worker,
         verified_client,
         miner_id,
         "deal1".to_string(),
-        deal_size,
+        PaddedPieceSize(deal_size),
         true,
         deal_start,
         deal_term_min,
```

Comment:
nit: When setting sector_term below it would be more clear to do so in terms of `MARKET_DEFAULT_ALLOCATION_TERM_BUFFER` or assert using this constant that the constraint referenced in the comment is met.